app/app.js

```diff
@@ -0,0 +1,96 @@
+const http = require('http');
+
+const hostname = '127.0.0.1';
```

Comment:
Best Practice: Hard coding server addresses and ports limits the portability of your code. This should be handled with an environmental variable, and default to 127... if the envar is not present.